hicommonwealth/commonwealth
# Updated skeleton components to use proper stylistic guidelines

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/4799

## Description of Changes
Now added separate skeleton component files for components that use skeleton loader

## "How We Fixed It"
By adding separate skeleton component files for components that use skeleton loader

## Test Plan
- Visit any discussion page, verify it shows a skeleton for thread cards and the reaction button also had a skeleton
- Visit any thread details page, verify it shows a skeleton for the layout
- Open any snapshot proposal card, verify it shows a skeleton until it loads
- Open the overview page and verify it shows a skeleton for the first time until data loads

## Deployment Plan
N/A

## Other Considerations
N/A
